setup.py

```diff
@@ -19,8 +19,16 @@
 __version__ = None
 exec(open('cirq/_version.py').read())
 
+description = ('A framework for creating, editing, and invoking '
+               'Noisy Intermediate Scale Quantum (NISQ) circuits.')
+
 # Readme file as long_description:
-long_description = io.open('README.rst', encoding='utf-8').read()
+long_description = ('====\n'
+                    'Cirq\n'
+                    '====\n')
+stream = io.open('README.rst', encoding='utf-8')
```

Comment:
Not really needed here but it would be nice to use a with statement to close the file.

Reply:
Acknowledged.